### DISCUSSION TITLE
Add row variables and selectors

### DISCUSSION THREAD

**Discussion by mjf** [grafana/grafana#47179] [Dashboards, Panels & Visualizations]
Hello team.

I propose to add variables we could assign to a particular (existing?) row (or multiple rows?) in the dashboard. Variable selectors of these variables would appear right under the particular row line (above the row content; the same way as the "global" variable selectors do now so that this part of code can be certainly reused).

**Comment by mckn** [comment]
@grafana/user-essentials

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
